### PR TITLE
S4eric - Update surefile file for 6 figures

### PR DESCRIPTION
**Pull request by erickain**
# WELCOME TO DEVOPS-EASY-LEARNING S3 PIPELINE  PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [x] Tests added and run
- [x] (External) Interfaces documented
- [x] Has security implications (describe below)

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

FROM HERE, LIFE AND  CAREER CHANGE ARE  POSSIBLE

### CODE REVIEW

File: 6-figures-Surefire [hunk 1]
```diff
@@ -0,0 +1,4 @@
+I am going to have multiple 6 figures job.
+Everything is possible 
+
```
Comment (devopseasylearning) [inline 6-figures-Surefire]: please remove this gap

**Comment by devopseasylearning** [comment]
@erickain the code is too fat

File: 6-figures-Surefire (hunk 1 above)
Comment (s4chidi) [reply 6-figures-Surefire]: It's very easy to resolve

**Comment by Tcedric2022** [comment]
good catch

File: 6-figures-Surefire (hunk 1 above)
Comment (anatolenga) [reply 6-figures-Surefire]: the gap is really big

Comment (jahelwilson) [reply 6-figures-Surefire]: It will make it lighter and simple

**Comment by s4serge** [comment]
good job

File: 6-figures-Surefire [hunk 2]
```diff
@@ -0,0 +1,4 @@
+I am going to have multiple 6 figures job.
+Everything is possible
```
Comment (gtoukam) [inline 6-figures-Surefire:2]: goog line

**Comment by s4nonon** [comment]
i aggre

**Comment by kayoucarolle** [comment]
great

**Comment by Rylogan** [comment]
Good job

**Comment by wounjiem** [comment]
Update not yet completed

File: 6-figures-Surefire (hunk 1 above)
Comment (Agnonyao) [reply 6-figures-Surefire]: it true you have to remote this gap. thanks

**Comment by clovisbernard** [comment]
The code is good

**Comment by Collince-T** [comment]
Good Job !

**Comment by erickain** [comment]
Making the changes.

 I thought it should be an improvement

**Comment by s4minata** [comment]
okay got it

**Comment by YvesT13** [comment]
keep going man !

**Comment by patriciakuissi** [comment]
yes good

**Comment by s4adelaide1** [comment]
i agree with Mr Eric, it is too fat

**Comment by s4olga** [comment]
i agree. its  possible

**Comment by Souleyhub** [comment]
SS

**Comment by krandou** [comment]
Is not really professional

**Comment by stivia14** [comment]
good catch

**Comment by s4diane** [comment]
great

**Comment by s4assan** [comment]
i htink thats good

**Comment by s4marthe** [comment]
the code is good for me

**Comment by devopseasylearning** [comment]
> SS

SS ?

**Comment by s4anthony** [comment]
good job ERIC

**Comment by anatolenga** [comment]
> the code is good for me

I believe it

**Comment by Agnonyao** [comment]
we got it thanks.

**Comment by s4olga** [comment]
> 

@

> Is not really professional

how so?

**Comment by ekus-little** [comment]
good job

**Comment by Rylogan** [comment]
This is interesting!

**Comment by Collince-T** [comment]
> yes good



> i agree with Mr Eric, it is too fat

@s4adelaide1 Good Job !

File: 6-figures-Surefire [hunk 3]
```diff
@@ -0,0 +1,3 @@
+I am going to have multiple 6 figures job.
+Everything is possible 
+Just register to devops Easy Learning
```
Comment (devopseasylearning) [inline 6-figures-Surefire:3]: the best choice you ever made

**Review by devopseasylearning — APPROVED** [review]

**Review by Souleyhub — COMMENTED** [review]
S is Souleymane 
S is for Sidibe

File: 6-figures-Surefire (hunk 1 above)
Comment (s4adelaide1) [reply 6-figures-Surefire]: i think the gap makes sens

Comment (s4adelaide1) [reply 6-figures-Surefire]: he plans to insert one information there.

File: 6-figures-Surefire (hunk 3 above)
Comment (s4adelaide1) [reply 6-figures-Surefire:3]: definetely!

**Review by s4marthe — APPROVED** [review]

**Review by kayoucarolle — COMMENTED** [review]
good

**Comment by clovisbernard** [comment]
Good job

File: 6-figures-Surefire (hunk 3 above)
Comment (s4serge) [reply 6-figures-Surefire:3]: i don't agree.

**Review by ekus-little — COMMENTED** [review]
continue with your good work

**Review by s4olga — COMMENTED** [review]
thanks

**Comment by clovisbernard** [comment]
> continue with your good work

**Pull request closed without merging**